dug_4_osfv.md

```diff
@@ -0,0 +1,214 @@
+class: center, middle, intro
+
+# &#x1F386; Dasharo Open Source Firmware Validation &#x1F386;
+
+.center[<img src="/remark-templates/dasharo-presentation-template/images/dasharo-sygnet-white.svg" width="150px" style="margin-left:-20px">]
+
+???
+
+---
+
+# Agenda
+
+* Introduction to Dasharo Open Source Firmware Validation (OSFV)
+* Exploring OSFV v0.2
+* Recent improvements
+* Getting started with OSFV
+* Upcoming Features: OSFV v0.3 and Beyond
+* Q&A
+
+???
+
+---
+
+# History
+
+.center.image-99[![](/img/osfv.png)]
+
+* We use OSFV at least since 2018, when we started validate PC Engines releases
+* Using those scripts we over **50k** tests publicly releasing **150+** binaries
+  based on open-source firmware.
+* Initially it was closed source because of assumption that validation provides
+  majority of the value in open-source firmware development.
+
+---
+
+# Dasharo OSVF
+
+.center[https://github.com/Dasharo/open-source-firmware-validation]
+
+* Dasharo Open Sourve Firmware Validation consist of MIT-licensed scripts
+  written in Robot Frameworks with the purpose of validation of open-source
+  firmware (mainly Dasharo).
+* Robot Framework is a generic open source automation framework. It can be used
+  for test automation and robotic process automation (RPA).
+    - Used by OpenBMC among other organizations.
+* Key Features of OSFV:
+    - **Hardware Compatibility Testing**: audio, cpu, fan, network, docking
+    stations, displays, network, thunderbolt, USB and more.
+    - **Performance and Stability Testing**: boot time, cpu frequency and
+    temperature, power cycle testing.
+    - **Dashro Security Features Testing**: UEFI Secure Boot, measured boot, TPM,
+    verified boot, TCG OPAL, ME, DMA protection and more.
+* To maximize use of Dasharo OSFV you need dedicated infrastructure.
```

Comment:
We should have some vision slides. What is the future of OSFV? How will it earn for itself?
There is some old stuff in internal Dasharo Overview training, and we have probably some PRs not merged related to the Dasharo Certification Program.

https://docs.dasharo.com/osf-trivia-list/dasharo/#what-is-dasharo-certification-program

https://github.com/Dasharo/docs/pull/337/files